doc/whats_new.rst

```diff
@@ -56,6 +56,7 @@ BUG
 API
 ~~~
 
+    - Refactored `GLM` class so that it is fully support by `scikit-learn` by `Titipat Achakulvisut`_.
```

Comment:
umm ... this is in the previous release. The current release is above.

I would put this under bug fix since you technically did not refactor anything. You added tests and checks to make sure everything worked smoothly with scikit-learn.